src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -99,23 +99,25 @@ export default function GamesSettings() {
 
         <AlternativeExe />
 
-        <ShowFPS />
+        {!nativeGame && <ShowFPS />}
 
-        <PreferSystemLibs />
+        {isLinux && !nativeGame && (
+          <>
+            <PreferSystemLibs />
 
-        <EnableFSR />
+            <EnableFSR />
 
-        <EnableEsync />
+            <EnableEsync />
 
-        <EnableFsync />
+            <EnableFsync />
 
-        <GameMode />
+            <GameMode />
+          </>
+        )}
 
         <UseDGPU />
 
-        <AudioFix />
```

Comment:
Was this intentional?

Reply:
Just commented on Discord that most distros now use PipeWire and I dont really know a case where some game need this. If so, the person can always use the Env variables, what do you think?

Reply:
Do we want to remove the actual `AudioFix` component as well then?

Reply:
yes, will do